Makefile

```diff
@@ -12,17 +13,25 @@ test:
 	$(MAKE) -C exporters/gcp_to_cwl/ test
 	$(MAKE) -C es_managers/es_idx_manager/ test
 
+.PHONY: plan-infrastructure
+plan-infrastructure:
+	./infrastructure.sh apply
```

Comment:
This should be a plan instead of an apply right?

Reply:
Oh true. This is garbo. Removing it.